main.py

```diff
@@ -297,6 +303,33 @@ async def calculate_withdrawal(interaction: discord.Interaction, withdrawal_date
             "An error occurred while calculating the withdrawal date. Please try again later."
         )
 
+@bot.tree.command(
+        name="lucky_winner", description="Randomly pick lucky winner(s)"
+)
+@app_commands.describe(
+    range="Range of numbers to choose from. Provided as `1-10`. Both numbers are included as possible winners.",
```

Comment:
I think asking ranges in two different parameters will be good as it will be an ease for user using mobile device too.